src/test/java/uk/gov/hmcts/cmc/claimstore/services/MediationCSVGeneratorTest.java

```diff
@@ -28,8 +28,10 @@
 
     private MediationCSVGenerator mediationCSVGenerator;
 
+    private static final String CARRIAGE_RETURN = "\r\n";
```

Comment:
Bit of a misnomer - only `\r` is a carriage return. `CRLF` is understandable, or just `NEW_LINE`